app/controllers/chefs_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ChefsController < ApplicationController
+    def show
+        @chef = Chef.find(params[:id])
+        @ingredients = @chef.dishes.flat_map(&:ingredients).uniq    end
```

Comment:
For this page, all we need is the chef name and then you can create a link to the ingredients page (which is handled from your chef_ingredients_controller. So, you don't need this line 4